packages/devtools-launchpad/webpack.config.js

```diff
@@ -50,18 +52,9 @@ module.exports = (webpackConfig, envConfig, options = {}) => {
   });
 
   webpackConfig.module.rules.push({
-    test: /\.svg$/,
```

Comment:
didn't look at the following yet, but if there are no changes in reps, I think that will break them.

Reply:
yep... reps will need to stay on the old launchpad for a bit. same w/ the dbg